src/sparseml/modifiers/obcq/utils/sparsegpt.py

```diff
@@ -18,7 +18,13 @@
 
 import torch
 import torch.nn as nn
-import transformers
+
+
+try:
+    import transformers
+except ImportError as err:
+    transformers = None
+    transformers_err = err
```

Comment:
I think this is also going to clash with a change I made, but I like this solution better! In pair programming today lets go through the merge conflicts

Reply:
updated!